Using-Mastodon/FAQ.md

```diff
@@ -1,253 +0,0 @@
-Frequently Asked Questions
```

Comment:
Adding the new file is good but you probably should not delete the original

It would also be nice to add a link at the top so that people know to find the spanish translation :)